man/ocitools-generate.1.md

```diff
@@ -137,18 +137,19 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
+**--poststart**=[CMD[:ARG]]
```

Comment:
On 08/17/2016 11:01 AM, W. Trevor King wrote:
> Why the brackets around the whole thing? I'd expect:
> 
> *--poststart*=CMD[:ARG...]
> 
brackets around the whole thing means it accepts more than one, it's a slice.
> *--poststart*=CMD[:ARG...]
looks like only ARG accepted as slice.